From 8feeb73427c7eaf075c587c856fe6f445422a897 Mon Sep 17 00:00:00 2001
From: kXuan <kxuanobj@gmail.com>
Date: Sat, 29 Jan 2022 22:19:46 +0800
Subject: [PATCH] remove harmful -static flag

The "-static" force g++ use static lib. But there is no static lib in the
protobuf package.

Signed-off-by: kXuan <kxuanobj@gmail.com>
---
 javascript/net/grpc/web/generator/Makefile | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/javascript/net/grpc/web/generator/Makefile b/javascript/net/grpc/web/generator/Makefile
index 2c30389..3cdc1e1 100644
--- a/javascript/net/grpc/web/generator/Makefile
+++ b/javascript/net/grpc/web/generator/Makefile
@@ -20,11 +20,6 @@ PREFIX ?= /usr/local
 MIN_MACOS_VERSION := 10.7 # Supports OS X Lion
 
 UNAME_S := $(shell uname -s)
-ifeq ($(UNAME_S),Darwin)
-  CXXFLAGS += -stdlib=libc++ -mmacosx-version-min=$(MIN_MACOS_VERSION)
-else ifeq ($(UNAME_S),Linux)
-  LDFLAGS += -static
-endif
 
 all: protoc-gen-grpc-web
 
-- 
2.34.1

